### DISCUSSION TITLE
zehn theme lag

### DISCUSSION THREAD

**Discussion by sakkuyaa** [yurisuika/Zehn#15] [Support] [answered]
the zehn theme can be laggy whenever I open steam up, and also when I click on something on my library and when I hover over my games.

**Comment by sakkuyaa** [comment]
and also, steam will go black and transparent sometimes whenever you open it.

**Comment by yurisuika** [comment]
There is a config option to change the blur radius that I expect should be one of the most pressing causes of any lag, as the skin uses backdrop blurs and transparency extensively. Beyond that, I am sorry that I do not have an answer for why this would be the case. I understand that Chromium is not very performant, so perhaps a theme as extensive as Zehn may not run to well on some lower end computers. If I had any idea what could be done about it, I would see to improve it. :(

**Reply by sakkuyaa** [reply]
I see. But I think the first time you start using Zehn, its gonna lag a lot. After two days of using it, its not lagging anymore. But it probably will for lower end PC's because of the stuff the skin uses like you mentioned. 
This is still a good theme though, I like it.

**Comment by yurisuika** [comment, marked as answer]
I believe this issue was fixed a while back, there were some broken CSS entries that were unearthed when remapping the skin to the changes brought about by valve obfuscating the classes some months back. I noticed a slight improvement myself, which was probably way worse on lower spec PCs.
